### PR TITLE
Chore: Add audit reports

### PR DESCRIPTION
**Pull request by marvinkruse**
**Status**
✅ Done, waiting for review...

---

Use the links below to see what it looks like:
In our main Readme, we link to our security guidelines, which contains a section about audits (see updated version [here](https://github.com/InverterNetwork/contracts/blob/add-audit-reports/SECURITY.md#security-guideline)). From there you are referred to see all audits, which leads you [here](https://github.com/InverterNetwork/contracts/blob/add-audit-reports/audits).

I've added the reports to the repo, so we'll always have them. The original links are still there though, so people can verify that these reports are real.

Any feedback?

### CODE REVIEW

**Comment by marvinkruse** [comment]
@Zitzak Updated with final Hats report (they fixed some formatting issues I found). So this PR is now ready and contains the final links + pdfs from the reports so they never disappear. Wdyt?

File: SECURITY.md [hunk 1]
```diff
@@ -10,24 +10,26 @@ If you discover a potential security vulnerability or issue within our smart con
...
 
 We are committed to transparency and will make the results of our security reviews and audits available to the public, fostering trust and confidence in the security of our platform.
 
+**You can find all of our audit reports [here](./audits/).**
+
+## Responsible Disclosure
+
+Please practice responsible disclosure when reporting security issues. We kindly request that you do not publicly disclose or discuss the vulnerability until we have had sufficient time to investigate and address it. Privately disclosing the issue to us via email allows us to work on a fix without putting our users' funds at risk.
+
+We understand the importance of acknowledging and rewarding security researchers for their efforts. Even though our bug bounty program is currently set up, we assure you that you will not be unpaid for your valuable contributions. We are committed to fairly compensating those who help us improve the security of our platform.
```
Comment (0xNuggan) [inline SECURITY.md]: I think  you meant: "Even though our **official** bug bounty program is currently **not** set up, we assure you that you will not be **left** unpaid for your valuable contributions."

Comment (marvinkruse) [reply SECURITY.md]: oh wow, yes... thank you!

Comment (marvinkruse) [reply SECURITY.md]: fixed!

File: audits/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Audits and Security Reviews
+
+At Inverter Network, we prioritize the security of our smart contracts and follow a rigorous process of security reviews and audits:
+
+* Per our development and security guidelines, we require an external review for each pull request (PR) that goes from the `dev` branch to the `main` branch in our repository. We have partnered with [Team Omega](https://teamomega.eth.limo) to conduct these security reviews directly in the PRs.
```
Comment (Zitzak) [inline audits/README.md:5]: Should we link the security reviews here as well?

**Review by Zitzak — APPROVED** [review]
I'm good with the way it is. We could, if we want, add the security review links. I'm doing the same within the docs. Only the last reviews done have not been through this process so we might not be able to link everything

File: audits/README.md (hunk 2 above)
Comment (marvinkruse) [reply audits/README.md:5]: You mean like I did in `audits/README.md`? @Zitzak

**Pull request merged**